addons/cloud_storage_google/tests/test_cloud_storage_google_attachment_controller.py

```diff
@@ -68,3 +69,34 @@ def test_cloud_storage_google_attachment_upload(self):
                     "upload_info": {"method": "PUT", "response_status": 200, "url": "[url]"},
                 },
             )
+
+    def test_mail_composer_cloud_storage_attachment(self):
+        """Ensure cloud attachments are converted to links in outgoing emails."""
+        self.env["ir.config_parameter"].set_param("cloud_storage_provider", "google")
+        self.env["ir.config_parameter"].set_param("cloud_storage_min_file_size", 1)
+
+        partner = self.env["res.partner"].create({"name": "Cloud Test Partner", "email": "cloud@test.com"})
+        cloud_attachment = self.env["ir.attachment"].create({
```

Comment:
Maybe also create another test for attachments
1. a large non cloud storage attachment
2. cloud stroage attachment

Check if the both attachment links are added and without duplicate.